plugins/woocommerce/client/admin/client/launch-your-store/hub/sidebar/components/mobile-header.tsx

```diff
@@ -0,0 +1,38 @@
+/**
+ * External dependencies
+ */
+import { Button } from '@wordpress/components';
+import { chevronLeft } from '@wordpress/icons';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import type { SidebarComponentProps } from '../xstate';
+import MobileSidebarToggle from './mobile-toggle';
+
+export const LaunchStoreHubMobileHeader = ( props: SidebarComponentProps ) => {
+	const handleBackClick = () => {
+		props.sendEventToSidebar( {
+			type: 'POP_BROWSER_STACK', // go back to previous URL
+		} );
+	};
+
+	return (
+		<div className="mobile-header launch-store-hub-mobile-header">
+			<Button
+				className="mobile-header__back-button"
+				onClick={ handleBackClick }
+				icon={ chevronLeft }
+				iconSize={ 20 }
+				aria-label={ __( 'Go back', 'woocommerce' ) }
+			/>
+			<h1 className="mobile-header__title">
+				{ __( 'Launch Your Store', 'woocommerce' ) }
```

Comment:
Use sentence-case everywhere:

```suggestion
				{ __( 'Launch your store', 'woocommerce' ) }
```